website/project/views/contributor.py

```diff
@@ -292,7 +292,7 @@ def project_manage_contributors(auth, node, **kwargs):
         )
         if node.is_public:
             return {'redirectUrl': node.url}
-        return {'redirectUrl': web_url_for('dashboard')}
+        return {'redirectUrl': web_url_for('my_projects')}
```

Comment:
Is this correct, or should it actually go to the dashboard? A question for @saradbowman or @nicipfeiffer I think. Since the default login location is the dashboard, should all default redirects be the dashboard? Is the dashboard our home page, essentially, while my_projects might be something that always has to be explicitly chosen? Or are there rules for what circumstances will direct you to the dashboard and what other circumstances take you to my_projects?

My preference is for always to the dashboard unless you click explicitly on my_projects (but my preference isn't the important one).